### PR TITLE
Use version properties in the top level pom.xml instead of literals

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling even more cosmetics

File: pom.xml [hunk 1]
```diff
@@ -253,27 +257,37 @@
...
+              project.properties.setProperty('uuid.hawkular.secretstore', uuidUi)
+            ]]></script>
+          </scripts>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${version.org.codehaus.groovy}</version>
+            <scope>runtime</scope>
```
Comment (jpkrohling) [inline pom.xml:288]: `runtime` dependencies are not included in the war, right? IIRC, it's `runtime` for the time that the build runs, not for the time the application runs, but better ask the maven expert :)

**Comment by jpkrohling** [comment]
Once the question about the `runtime` is cleared, this can be merged.

File: pom.xml (hunk 1 above)
Comment (ppalaga) [inline pom.xml:288]: This is just a plugin execution dependency that has nothing to do with the `war`.

**Pull request merged**